clusterpools/templates/install-config.azure.yaml.template

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+metadata:
+  name: 
+baseDomain: 
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  replicas: 3
+  platform:
+    azure:
+      osDisk:
+        diskSizeGB: 128
+      type:  Standard_D4s_v3
+compute:
+- hyperthreading: Enabled
+  name: worker
+  replicas: 3
+  platform:
+    azure:
+      type:  Standard_D2s_v3
+      osDisk:
+        diskSizeGB: 128
+      zones:
+      - "1"
+      - "2"
+      - "3"
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineCIDR: 10.0.0.0/16
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  azure:
+    baseDomainResourceGroupName:  __CLUSTERPOOL_BASE_DOMAIN_RESOURCE_GROUP_NAME__
+    region: __CLUSTERPOOL_AZURE_REGION__
+pullSecret: "" # skip, hive will inject based on it's secrets
```

Comment:
it's --> its

Reply:
@dhaiducek I don't know how to say it but I stole this install-config from the RHACM product's UI page so we have it wrong in the product!

Reply:
PANIC!!!